### ISSUE TITLE
Chrome warning on SameSite attribute 

### ISSUE THREAD

**Issue by ocherfas** [grafana/grafana#19847]
**What happened**:
We embed grafana UI in our own app and configure grafana instance with oauth authentication.
To enable such embedding, we configure the grafana instance with:
########
[security]
cookie_samesite = none
##########

While doing so, chrome issues a warning:
A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.

Furthermore, when enabling the flag 'SameSite by default cookies' in chrome the oauth login fails.

From my understanding because of a bug in safari (see issue #18332) when configuring same_site = none the attribute SameSite is dropped from the cookie, but this will cause an issue on future chrome version 80, when this flag will be enabled by default.

**What you expected to happen**:
Login with oauth would succeed when flag 'SameSite by default cookies' in chrome is enabled.

**How to reproduce it (as minimally and precisely as possible)**:
- Configure grafana to use oauth mechanism.
- Embed grafana UI in another app.
- Enable chrome 'SameSite by default cookies' flag.
- Try to login into grafana from the embedded iframe.

**Environment**:
- Grafana version: v6.3.6
- Data source type & version:
- OS Grafana is installed on: GrafanaLabs
- User OS & Browser: MacOS. Chrome 77.0.3865.120
- Grafana plugins:
- Others:

**Comment by marefr** [comment]
Thanks for the detailed report. The solution to this change in Chrome and the Safari bug is probably #18456 so I'll close this. Will add a reference back to this issue.

**Comment by marefr** [comment]
In second thought, reopening this since there may be more issues related to this if you have cookie secure enabled and samesite none configured.

**Comment by schoolpost** [comment]
I'm having the same issue, I'm embedding grafana into a web application. ( iframe ) 

My usage is development purposes only, with that I'm using simple anonymous view-only use of grafana ( no auth ) will also add the grafana instance is on a different domain from the development machine. ( but I don't figure this is the cause for the issue ) 

I've set the samesite value in the grafana.ini to all of the 3 possible values, nothing stops the chrome warning. I've also disabled the samesite flag within chrome and still the issue persists. 

This warning isn't an issue on its own, the resulting issue of this samesite warning Is that it BLOCKS any of the javascript in the html page from even being loaded. ( not visible in the html ) 

searching around nothing describes what is blocking my javascript,
